pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -283,7 +278,7 @@ func (r *CloneReconcilerBase) doCrossNamespaceClone(log logr.Logger,
 	return nil, nil
 }
 
-func (r *CloneReconcilerBase) initTransfer(log logr.Logger, syncRes *dataVolumeCloneSyncResult, name string) (bool, error) {
```

Comment:
you have `GetCloneSourceNameAndNamespace()` so why add namespace param here and in `doCrossNamespaceClone()`?

Reply:
I only call `GetCloneSourceNameAndNamespace` when we don't know for sure,
which is not the case in `doCrossNamespaceClone` which gets called from a pvc/snap clone controller